packages/zosfiles/src/cli/create/ussDir/uss.options.ts

```diff
@@ -0,0 +1,35 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandOptionDefinition } from "@zowe/imperative";
+
+import i18nTypings from "../../-strings-/en";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../../-strings-/en").default as typeof i18nTypings).CREATE.ACTIONS.USSDIR.OPTIONS;
+
+/**
+ * Specifies the file or directory permission bits to be used in creating the file or directory.
+ */
+export const UssCreateOptions: { [key: string]: ICommandOptionDefinition } = {
+    /**
+     * The permission bits for owner, group, and world users of the directory
+     * @type {ICommandOptionDefinition}
+     */
+    mode: {
+        name: "mode",
+        aliases: ["m"],
+        description: strings.MODE,
+        type: "string",
+        // tslint:disable-next-line: no-magic-numbers
+        stringLengthRange: [9, 9]
```

Comment:
I don't think I've ever `stringLengthRange` specified this way... it makes sense if it works 😄 ... have we verified that it does?

Reply:
Confirmed that this works, produces the following error message when value has incorrect length:
```
Syntax Error:
Invalid value length for option:
--mode

You specified a string of length 3:
abc

The length must be between 9 and 9 (inclusive)
```

Reply:
This seems unexpected though, perhaps custom validation should be added to enforce that the mode is valid when it is 9 characters long:
```
$ zowe files create uss-directory //a/johti04/test --mode abcabcabc
USS file or directory created successfully.
```

Reply:
That is strange. I'm guessing the z/OSMF API doesn't reject a invalid values. I think we should document this in an issue when this is merged. We can later determine how to handle it?

Reply:
I just show it have option for stringLengthRange defined, so use it. Also I tested in my local it should be work.